src/Process/Pool/Logger/Message.php

```diff
@@ -17,14 +18,11 @@ class Message implements MessageInterface, \JsonSerializable
 
     protected $time;
     protected $level;
-    protected $process_id;
-    protected $process_path;
```

Comment:
I was actually thinking we would leave these in for now but mark them as deprecated. Changing the contract of the log messages seems pretty significant

Reply:
Restored with `@deprecated` flag.